### DISCUSSION TITLE
[BUG] It does not allow you to put the same start and expiration day without hours in the task list.

### DISCUSSION THREAD

**Discussion by jonathanathe** [TechbeeAT/jtxBoard#921] [Ideas]
**Describe the bug**

Allow adding the same date (without putting the times) of "Started" and "Expiration". And that the error of "The due date cannot be earlier than the start date" does not come out since that is not correct, it is not previous is the same this hinders to be able to add quick tasks that you need to do during the day and that do not have a start and end time is only during the day. **Describe the bug** Allow adding the same date (without putting the times) of "Started" and "Expiration". And that the error of "The due date cannot be earlier than the start date" does not come out since that is not correct, it is not previous is the same this hinders to be able to add quick tasks that you need to do during the day and that do not have a start and end time is only during the day.

**To Reproduce**
1. We enter to create a new task.
2. We try to put the same date without time in Started and Expiration. 

**Expected behavior**
I hope to be able to add quick tasks in which I can put the start and due date on the same day without having to put the hours and do not make me mistake that the due date is earlier than the start. 
**Screenshots**

![image](https://github.com/TechbeeAT/jtxBoard/assets/34164094/1f5186a2-4f1b-4688-a912-57d70b76f9cc)

**Device and version:**
 - Device: Samsung Galaxy S223 Ultra
 - Android version: Android 13
 - jtx Board version: 2.05.06.gplay
 - Downloaded from: Play Store

**Comment by patrickunterwegs** [comment]
Hi @jonathanathe !
This is a basic problem in the definition of the standard:

```
Description:  This property defines the date and time before which a
      to-do is expected to be completed.  For cases where this property
      is specified in a "VTODO" calendar component that also specifies a
      "DTSTART" property, the value type of this property MUST be the
      same as the "DTSTART" property, and the value of this property MUST be later in time than the value of the "DTSTART" property.
```


Now some people still interpret this that an entry can have a start and due date at the same time. But many servers have a strict interpretation of this and would throw an error if an app allows this constellation. 

As a developer I can only choose between complaints, that this date combination is not possible or complaints, that servers throw an error when that constellation is allowed...

**Comment by patrickunterwegs** [comment]
I'll put this topic into discussions to keep it there open and to see if maybe others come up with ideas.
